### PR TITLE
feat(action): Make commit range configurable

### PR DESCRIPTION
**Pull request by orhun**
Addresses #320

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -2,11 +2,17 @@ name: "committed-action"
 author: "Ed Page"
 description: "Run committed to check commit history"
 
-inputs: {}
+inputs:
+  commits:
+    description: "commit range to check"
+    required: false
+    default: "HEAD~..HEAD^2"
```
Comment (epage) [inline action.yml:9]: I'd like to resolve the conversation in #320 first

File: docker/entrypoint.sh [hunk 2]
```diff
@@ -17,12 +18,12 @@ COMMAND="${CMD_NAME}"
...
+git log --color=always --graph --oneline "${commits}" || true
 echo ""
 echo "Against 'committed.toml':"
 ${COMMAND} --dump-config - || true
 echo ""
 echo "If this fails, don't sweat it. We're trying to encourage clear communication and not hinder contributions."
 echo "If it is a reasonable issue and you lack time or feel uncomfortable fixing it yourself,"
 echo "let us know and we can mentor or fix it."
-${COMMAND} --color=always -vv --no-merge-commit HEAD~..HEAD^2
+${COMMAND} --color=always -vv "${commits}"
```
Comment (epage) [inline docker/entrypoint.sh:29]: Why was `--no-merge-commit` removed?  How do we plan to handle that case?

Comment (epage) [reply docker/entrypoint.sh]: Could you instead swap it around to not be breaking change?

Comment (orhun) [reply docker/entrypoint.sh]: Good point! Done in 8452a4248c4910fe599b9b72c6989a7508171084

**Pull request merged**